timeless/reservations/views.py

```diff
@@ -76,13 +78,16 @@ def list_reservations(reservations):
 
 
 @bp.route("/create", methods=("GET", "POST"))
-def create(reservation):
-    if request.method == "POST":
-        flash("Create not yet implemented")
-
+def create():
+    """ Create new reservation """
+    form = ReservationForm(request.form)
+    form.validate()
+    if request.method == "POST" and form.validate():
+        form.save()
+        return redirect(url_for("reservations.list_reservations"))
```

Comment:
@Vatavuk Can we return ``BAD_REQUEST`` in an ``else``  here? That's how it was done in TableShape.edit() for instance. And it make more sense to tell the user that they did something wrong rather than simply listing the existing tables.

Reply:
@amihaiemil I think this is a wrong approach, we should return a template containing errors to show the user validation problems directly on the browser. That's common practice (at least to my knowledge). We can render the errors in template using {{form.errors}}. I think I will raise an issue for this. If you don't agree we can ask `ARC` for opinion.

Reply:
@Vatavuk 
> I think I will raise an issue for this.

Makes sense, I think you can do that and discuss with the ARC there :)